apps/src/blockly/customBlocks/cdoBlockly/spritelabBlocks.js

```diff
@@ -233,6 +233,8 @@ export const blocks = {
       ];
     };
 
+    // This is only used by CDO Blockly. When we are ready to remove support
+    // for CDO Blockly we can remove this call.
```

Comment:
This entire file is only used by CDO Blockly so the statement really extends to the entire file as well.

Reply:
ah good point, will update